fix: remove ending periods from non-sentence list items (#23845)
mdn/content: files/en-us/web/api/htmlcanvaselement/height/index.md

## See also

> - The interface defining it, {{domxref("HTMLCanvasElement")}}

Reviewer: ```suggestion
- {{domxref("HTMLCanvasElement")}}: Interface used to define the `HTMLCanvasElement.height` property
```
would be nice to replace the pronoun 'it' to be more specific

> - The other property controlling the size of the canvas,

Reviewer: ```suggestion
- {{domxref("HTMLCanvasElement.width")}}: Other property used to control the size of the canvas
```

>   {{domxref("HTMLCanvasElement.width")}}

Reviewer: ```suggestion

```
